go_repositories.bzl

```diff
@@ -0,0 +1,345 @@
+load("@bazel_gazelle//:deps.bzl", "go_repository")
```

Comment:
Is this gazelle generated? Must be. Wish gazelle added a header to warn people it's auto-generated.

Reply:
Yeah PR description says this is gazelle generated. Wish it had a header.